### PR TITLE
Support setting 'Per Site Scaling' on AppService plan create

### LINKED ISSUE #7888: [webapp] - App Service support high density per-app scaling
It doesn't seem that the `az appservice plan create` command offers support to enable the "high density" "PerSiteScaling" feature described [here](https://docs.microsoft.com/en-us/azure/app-service/app-service-high-density-hosting)

It would be great if this support could be added. It would also necessary to support setting the number of workers on a web app basis (perhaps with `az webapp config set`)

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes #7888
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -83,6 +83,10 @@ def load_arguments(self, _):
         c.argument('sku', arg_type=sku_arg_type)
         c.argument('is_linux', action='store_true', required=False, help='host web app on Linux worker')
         c.argument('hyper_v', action='store_true', required=False, help='Host web app on Windows container', is_preview=True)
+        c.argument('per_site_scaling', action='store_true', required=False, help='Enable per-app scaling at the '
+                                                                                 'App Service plan level to allow for '
+                                                                                 'scaling an app independently from '
+                                                                                 'the App Service plan that hosts it.')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py]: Maybe you can put help= in a new line.

**Comment by qwordy** [comment]
Update HISTORY.rst. Thanks.

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 2]
```diff
@@ -205,6 +209,7 @@ def load_arguments(self, _):
             c.argument('allowed_origins', options_list=['--allowed-origins', '-a'], nargs='*', help='space separated origins that should be allowed to make cross-origin calls (for example: http://example.com:12345). To allow all, use "*" and remove all other origins from the list')
 
         with self.argument_context(scope + ' config set') as c:
+            c.argument('number_of_workers', options_list=['--number-of-workers'], help='The number of workers to be allocated.', type=int)
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py]: options_list can be ommited

Comment (panchagnula) [reply src/azure-cli/azure/cli/command_modules/appservice/_params.py]: thanks for the quick PR on this.

**Pull request merged**